cluster/common.sh

```diff
@@ -311,10 +311,9 @@ function set_binary_version() {
 function find-tar() {
   local -r tarball=$1
   locations=(
-    "${KUBE_ROOT}/bazel-bin/release/${tarball}"
```

Comment:
It seems that this is reverting CCM specific code change, there are some other. I'm doing upgrade to newest release-1.23 k/k /cluster directory to identify all differences between k/k and cloud-provider-gcp differences in #341